narwhals/_pandas_like/expr.py

```diff
@@ -48,6 +48,32 @@
 }
 
 
+def window_kwargs_to_pandas_equivalent(
+    function_name: str, kwargs: dict[str, object]
+) -> dict[str, object]:
+    unsupported_reverse_msg = (
+        "Cumulative operation with `reverse=True` is not supported in "
+        "over context for pandas-like backend."
+    )
+    if function_name == "shift":
+        pandas_kwargs: dict[str, object] = {"periods": kwargs["n"]}
+    elif function_name == "rank":
+        _method = kwargs["method"]
```

Comment:
just out of curiosity, would it be possible that the "method" key (or "descending") is not present?

Reply:
no, we always pass it